### PR TITLE
Fixed debug run for mac.

### PR DESCRIPTION
**Pull request by Frozen**
### Unit Test Coverage

Before:

```
<!-- copy/paste 'go test -cover' result in the directory you made change -->
```

After:

```
<!-- copy/paste 'go test -cover' result in the directory you made change -->
```

## Operational Checklist

1. **Does this PR introduce backward-incompatible changes to the on-disk data structure and/or the over-the-wire protocol?**. (If no, skip to question 8.)

    **YES|NO**

2. **Describe the migration plan.**. For each flag epoch, describe what changes take place at the flag epoch, the anticipated interactions between upgraded/non-upgraded nodes, and any special operational considerations for the migration.

3. **Describe how the plan was tested.**

4. **How much minimum baking period after the last flag epoch should we allow on Pangaea before promotion onto mainnet?**

5. **What are the planned flag epoch numbers and their ETAs on Pangaea?**

6. **What are the planned flag epoch numbers and their ETAs on mainnet?**

    Note that this must be enough to cover baking period on Pangaea.

7. **What should node operators know about this planned change?**

8. **Does this PR introduce backward-incompatible changes *NOT* related to on-disk data structure and/or over-the-wire protocol?** (If no, continue to question 11.)

    **YES|NO**

9. **Does the existing `node.sh` continue to work with this change?**

10. **What should node operators know about this change?**

11. **Does this PR introduce significant changes to the operational requirements of the node software, such as >20% increase in CPU, memory, and/or disk usage?**

### CODE REVIEW

**Review by ONECasey — APPROVED** [review]

**Comment by sophoah** [comment]
hey @Frozen what was the error message you had so you don't need to use sudo on your mac with that PR

File: test/kill_node.sh [hunk 1]
```diff
@@ -3,9 +3,15 @@
 # list of process name to be killed
 targetProcess=( "harmony" "bootnode" "soldier" "commander" "profiler" )
 
+NAME="$(uname -s)"
+
 for pname in "${targetProcess[@]}"
 do
-  sudo pkill -9 -e "^(${pname})$"
+  if [ $NAME == "Darwin" ]; then
+    pkill -9 "^(${pname})$"
```
Comment (sophoah) [inline test/kill_node.sh:11]: so you removed the sudo here, what was the error you had before that ? i assume on your mac your user account has root privilege ? is that the same for all mac OS version ?

Comment (Frozen) [reply test/kill_node.sh:11]: sudo requires password, but pkill works without it. 

> is that the same for all mac OS version ?
I don't really know, but more likely i have default settings.

File: test/kill_node.sh [hunk 2]
```diff
@@ -3,9 +3,15 @@
...
+  else
+    sudo pkill -9 -e "^(${pname})$"
```
Comment (MaxMustermann2) [inline test/kill_node.sh:13]: I think this was introduced by @GheisMohammadi but it should be removed. Rather, the user should call `sudo ./test/kill_node.sh` instead. 

```suggestion
    pkill -9 -e "^(${pname})$"
```

**Review by MaxMustermann2 — APPROVED** [review]

File: test/kill_node.sh (hunk 2 above)
Comment (ONECasey) [reply test/kill_node.sh:13]: @Frozen can you please change before we merge?

Comment (Frozen) [reply test/kill_node.sh]: fixed

**Pull request merged**